tests/wpt/mozilla/tests/mozilla/interfaces.worker.js

```diff
@@ -101,6 +101,7 @@ test_interfaces([
   "Touch",
   "TouchEvent",
   "TouchList",
+  "TransitionEvent",
```

Comment:
There should be no need to expose this interface to workers, so let's remove this.